packages/sui-changelog/README.md

```diff
@@ -0,0 +1,63 @@
+# sui-changelog
+
+> CLI to retrieve a changelog from a set of dependencies.
+
+## Description
+
+The main aim of this package is to build a changelog file with all the last changes you get in your last installation within your project. Dependencies are retrieved by its scope in the `node_modules` folder, so you can customize the scopes to get the changelog data from.
+
+## Usage
+
+Install `node@11` with `npm@6` in your env and run the following CLI command:
+
+```sh
+$ npx @s-ui/changelog
+```
+
+Or, if you already have it installed:
+
+```sh
+$ sui-changelog
+```
+
+### Adding more scopes
+
+By default, `sui-changelog` works only with `@s-ui` scoped packages, so if you want to add more scopes to your changelog, just add them in your project `package.json` file, the same way as the example below:
+
+```js
+{
+  "sui-changelog": {
+    "scopes": ["@my-awesome-scope", "@another-scope"]
+  }
+}
+```
+
+### Retrieving data from private repositories
+
+If you know you have some private respositories inside your set of dependencies, you should add a GitHub access token in order to make it work. Such token has to be added like the following example:
+
+```js
+{
+  "sui-changelog": {
+    "githubToken": "MY_AWESOME_GITHUB_PERSONAL_ACCESS_TOKEN"
```

Comment:
shouldn't this be passed by as environment var or cli argument? We should try to avoid having plain text tokens written in our repos

Reply:
@jcger and @carlosvillu You're right, this is not a good way of working with secrets, but at the end of the day, with any of these methods, we'll have the JARVIS token written in all our private repos. Just to make it clear, with your suggestion, this bin script will be called from a NPM script like this:
```json
{
  "scripts": {
    "changelog": "sui-changelog GITHUB_TOKEN=1234abcd"
  }
}
```

Reply:
I mean, you have to call without define in the command any env var. And later inside your code check is this vars exist.
https://github.com/SUI-Components/sui/blob/master/packages/sui-ssr/bin/sui-ssr-release.js#L46-L50

Reply:
Clear! It's already applied the changes you suggest 😉